src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/utils/TaskMapper.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.utils;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.camunda.CamundaObjectMapper;
+import uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.camunda.CamundaTask;
+import uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.camunda.CamundaVariable;
+import uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.camunda.Task;
+
+import java.util.Map;
+import java.util.Optional;
+
+
+@SuppressWarnings("PMD.UnnecessaryConstructor")
+@Service
+public class TaskMapper {
+
+
+    private final CamundaObjectMapper camundaObjectMapper;
+
+    @Autowired
+    public TaskMapper(CamundaObjectMapper camundaObjectMapper) {
+        this.camundaObjectMapper = camundaObjectMapper;
+    }
+
+    public Task mapToTaskObject(Map<String, CamundaVariable> localVariableResponse, CamundaTask camundaTask) {
+        // Camunda Attributes
+        String id = camundaTask.getId();
+        String name = camundaTask.getName();
+        String task = camundaTask.getFormKey();
+        String createdDate = camundaTask.getCreated().toString();
+        String dueDate = camundaTask.getDue().toString();
+        String assignee = camundaTask.getAssignee();
+        // Local Variables
+        String taskState = getValueFromObject(localVariableResponse.get("taskState"), String.class);
+        String securityClassification = getValueFromObject(localVariableResponse
+                                                               .get("securityClassification"), String.class);
+        String taskTitle = getValueFromObject(localVariableResponse.get("title"), String.class);
+        String executionType = getValueFromObject(localVariableResponse.get("executionType"), String.class);
+        boolean autoAssigned = false;
+        String taskSystem = getValueFromObject(localVariableResponse.get("taskSystem"), String.class);
+        String jurisdiction = getValueFromObject(localVariableResponse.get("jurisdiction"), String.class);
```

Comment:
Following on from the earlier comment about calling the /variables endpoint could we also rename these to what it was:

`getVariableValue(variables.get("ccdId"), String.class);`